code/game/mecha/equipment/tools/mining_tools.dm

```diff
@@ -135,6 +135,17 @@
 		if(target_part && prob(10 * drill_level))
 			target_part.dismember(BRUTE)
 
+/obj/item/mecha_parts/mecha_equipment/drill/experimental
+	name = "experimental exosuit drill"
+	desc = "An experimental exosuit drill, equipped with hydraulic system and made in light-green colors."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "corrupted_drill"
+	equip_cooldown = 5
+	drill_delay = 2
+	drill_level = DRILL_HARDENED
+	force = 30
```

Comment:
you realize this is the force from hitting someone with the item itself, correct?